homeassistant/components/trend/config_flow.py

```diff
@@ -0,0 +1,106 @@
+"""Config flow for Trend integration."""
+from __future__ import annotations
+
+from collections.abc import Mapping
+from typing import Any, cast
+
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import BinarySensorDeviceClass
+from homeassistant.components.sensor import DOMAIN as SENSOR_DOMAIN
+from homeassistant.const import (
+    CONF_ATTRIBUTE,
+    CONF_DEVICE_CLASS,
+    CONF_ENTITY_ID,
+    CONF_NAME,
+    UnitOfTime,
+)
+from homeassistant.helpers import selector
+from homeassistant.helpers.schema_config_entry_flow import (
+    SchemaCommonFlowHandler,
+    SchemaConfigFlowHandler,
+    SchemaFlowFormStep,
+)
+
+from .const import (
+    CONF_INVERT,
+    CONF_MAX_SAMPLES,
+    CONF_MIN_GRADIENT,
+    CONF_SAMPLE_DURATION,
+    DOMAIN,
+)
+
+NONE_SENTINEL = "none"
+
+
+async def get_options_schema(handler: SchemaCommonFlowHandler) -> vol.Schema:
+    """Get options schema."""
+    return vol.Schema(
+        {
+            vol.Optional(CONF_ATTRIBUTE): selector.AttributeSelector(
+                selector.AttributeSelectorConfig(
+                    entity_id=handler.options[CONF_ENTITY_ID]
+                )
+            ),
+            vol.Optional(CONF_DEVICE_CLASS): selector.SelectSelector(
```

Comment:
@emontnemery Can you give your opinion on having device class in the options flow as there was this discussion today for another helper having device class configurable whilst that was possible via the entity registry.

Reply:
Oh yeah right, then IMO this should be removed.

Reply:
Right, we should not have this in the options flow 👍

Reply:
I've removed the device class from the options flow now.